src/disco/css/App.scss

```diff
@@ -43,6 +51,25 @@ header {
   }
 }
 
+.cta {
+  align-self: center;
+  padding: 1em 1.5em;
+  margin: 30px 0;
+  background: $cta-background-color;
+  color: #fff;
+  text-decoration: none;
+  font-size: 13px;
+  transition: background .2s;
```

Comment:
I prefer using `200ms` when it's less than a second, but it probably doesn't matter. I think we wanted the leading zero for decimals, though?